modules/helm-installing-a-helm-chart-on-an-openshift-cluster.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/helm/getting-started-with-helm-on-openshift.adoc
+
+[id="installing-a-helm-chart-on-an-openshift-cluster_{context}"]
+
+= Installing a Helm chart on an {product-title} cluster
+
+.Prerequisites
+* You have a running {product-title} cluster and you have logged into it.
+* You have installed Helm.
+
+.Procedure
+
+. Create a new project:
++
+----
+$ oc new-project mysql
+----
+
+. Add a repository of Helm charts to your local Helm client:
++
+----
+$ helm repo add stable https://kubernetes-charts.storage.googleapis.com/
+"stable" has been added to your repositories
```

Comment:
@boczkowska we need to `helm repo update` after `helm repo add` to update repo index locally.